school/person.rb

```diff
@@ -24,4 +25,8 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
```

Comment:
- [ ] it is rental of a book. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. something like this
```ruby
def rent_book(book, date)
    Rental.new(date, book, self)
  end
```